IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -22,31 +22,31 @@
     <hr />
     <div>
         <table border="1" width="90%">
-            <tr>
-                <th>ID</th>
-                <th>Name</th>
-                <th>Brewery Type</th>
-                <th>Street</th>
-                <th>Address 1st Part</th>
-                <th>Address 2nd Part</th>
-                <th>City</th>
-                <th>State</th>
-                <th>County Province</th>
-                <th>Postal Code</th>
-                <th>Country</th>
-                <th>Longitude</th>
-                <th>Latitude</th>
-                <th>Phone</th>
-                <th>Website Url</th>
-                <th>Update At</th>
-                <th>Created At</th>
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Brewery Type</th>
+                    <th>Street</th>
+                    <th>Address 1st Part</th>
+                    <th>Address 2nd Part</th>
+                    <th>City</th>
+                    <th>State</th>
+                    <th>County Province</th>
+                    <th>Postal Code</th>
+                    <th>Country</th>
+                    <th>Longitude</th>
+                    <th>Latitude</th>
+                    <th>Phone</th>
+                    <th>Website Url</th>
+                    <th>Update At</th>
+                    <th>Created At</th>
+                </tr>
+            </thead>
 
-            </tr>
-            @{
-                foreach (Breweries.BreweryCollection brr in brew)
+            <tbody>
+                @foreach (Breweries.BreweryCollection brr in brew)
                 {
                     <tr>
-                        <td>@brr.Id</td>

```

Comment:
I removed this Id element from both the tables as because they are internal to the dataset being displayed. From security point of view its not safe to display Ids on the HTML page. Also, displaying Id will not benefit the user of the website.